sequencing/clinical_annotator/dxapp.json

```diff
@@ -0,0 +1,128 @@
+{
+  "name": "clinical_annotator",
+  "title": "Clinical Variant Annotator",
+  "summary": "Annotates a VCF File with most recent ClinVar TSV file and with a user provided HGMD PRO VCF file.",
+  "dxapi": "1.0.0",
+  "version": "1.0.0",
+  "inputSpec": [
+    {
+      "name": "variants_vcfgz",
+      "label": "VCF File(s)",
+      "class": "array:file",
+      "optional": false,
+      "patterns" : ["*.vcf.gz"],
+        "help": "The VCF File to be annotated (chr prefix must not be present in VCF)."
+    },
+    {
+      "name": "variant_summary",
+      "label": "Variant Summary File",
+      "class": "file",
+      "optional": false,
+      "patterns": ["*.txt.gz"],
+      "help": "A text file containing summarization of variant annotations (one such file can be downloaded here: ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/variant_summary.txt.gz)."
```

Comment:
A text file containing a summary of variant annotations. One such file can be downloaded here: ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/variant_summary.txt.gz.